Libplanet.Explorer/GraphTypes/TransactionType.cs

```diff
@@ -54,6 +55,14 @@ public TransactionType()
                 name: "Actions",
                 description: "A list of actions in this transaction."
             );
+            Field<NonNullGraphType<StringGraphType>>(
+                name: "SignedTx",
```

Comment:
How about renaming it to `SerializedPayload` to avoid confusion with `Signature`?

Reply:
Sure! I think that's a better name 👍